blueprints/ember-cli-notifications/index.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   // Use config to determine whether Font Awesome is imported into consuming app
   afterInstall: function(app) {
     var projectConfig = this.project.config(app.env);
-    var config = projectConfig['ember-cli-notifications'];
+    var config = projectConfig['ember-cli-notifications'] ||
+                 app.options['ember-cli-notifications'] ||
+                 { includeFontAwesome: false };
 
-    if (!config || config.icons !== 'bootstrap') {
+    if (config.includeFontAwesome) {
```

Comment:
We shouldn't add font-awesome into the consuming application's Bower dependencies if the config is set to use Bootstrap flavour icons. 

```if (config.includeFontAwesome && config.icons !== 'bootstrap')```